domains/jamestheflash22.json

```diff
@@ -1,11 +1,11 @@
 {
-  "description": "Use for my server description: Palette Of Pages",
+  "description": "Use for zoho mail",
   "repo": "https://github.com/JamesTheFlash22/JamesTheFlash22.github.io",
   "owner": {
     "username": "JamesTheFlash22",
     "email": "davidebu43@gmail.com"
   },
   "record": {
-    "CNAME": "jamestheflash22.github.io"
+    "CNAME": "zoho-verification=zb32026957.zmverify.zoho.eu"
```

Comment:
CNAMEs cannot contain the `=` character. I believe this is meant to be a TXT record.

Reply:
Hiw should i add the txt record?